src/deadline/job_attachments/asset_sync.py

```diff
@@ -313,6 +326,41 @@ def copied_download(
             else:
                 raise
 
+    def _check_and_write_local_manifests(
+        self, merged_manifests_by_root: dict[str, BaseAssetManifest], manifest_write_dir: str
+    ) -> list[str]:
+        """Write manifests to the session directory and check disk capacity is sufficient for the assets
```

Comment:
nit: I think this function could write manifests in locations other than the session directory, right? The location `manifest_write_dir` is a parameter. The docstring is a bit confusing because of that, I was looking for where it gets the session directory.